tgui/packages/tgui/interfaces/StripMenu.tsx

```diff
@@ -0,0 +1,394 @@
+                  let tooltip;
+
+                  if (item === null) {
+                    tooltip = slot.displayName;
+                  } else if ('name' in item) {
+                    alternateAction = ALTERNATE_ACTIONS[item.alternate];
+
+                    content = (
+                      <Box
+                        as="img"
+                        src={`data:image/jpeg;base64,${item.icon}`}
+                        height="100%"
+                        width="100%"
+                        style={{
+                          '-ms-interpolation-mode': 'nearest-neighbor',
+                          'vertical-align': 'middle',
+                        }}
+                      />
+                    );
+
+                    tooltip = item.name;
+                  } else if ('obscured' in item) {
+                    content = (
+                      <Icon
+                        name={
+                          item.obscured === ObscuringLevel.Completely
+                            ? 'ban'
+                            : 'eye-slash'
+                        }
+                        size={3}
+                        ml={0}
+                        mt={1.3}
+                        style={{
+                          'text-align': 'center',
+                          height: '100%',
+                          width: '100%',
+                        }}
+                      />
+                    );
+
+                    tooltip = `obscured ${slot.displayName}`;
+                  }
+
+                  return (
+                    <Stack.Item
+                      key={key}
+                      style={{
+                        width: BUTTON_DIMENSIONS,
+                        height: BUTTON_DIMENSIONS,
+                      }}>
+                      <Box
+                        style={{
+                          position: 'relative',
+                          width: '100%',
+                          height: '100%',
+                        }}>
+                        <Button
+                          onClick={() => {
+                            act('use', {
+                              key: keyAtSpot,
+                            });
+                          }}
+                          fluid
+                          tooltip={tooltip}
+                          style={{
+                            background: item?.interacting
+                              ? 'hsl(39, 73%, 30%)'
+                              : undefined,
+                            position: 'relative',
+                            width: '100%',
+                            height: '100%',
+                            padding: 0,
+                          }}>
+                          {slot.image && (
+                            <Box
+                              as="img"
+                              src={resolveAsset(slot.image)}
+                              opacity={0.7}
+                              style={{
```

Comment:
Can we migrate from embedded style options to scss? It should help simplify the code

Reply:
Where do I define the styles :sob:

Reply:
tgui/packages/tgui/styles/interfaces